### ISSUE TITLE
RHEL 8.2 image tests are failing

### ISSUE THREAD

**Issue by ondrejbudai** [osbuild/osbuild-composer#585]
The image tests are failing because of:

```
exec of "qemu-system-x86_64" failed: No such file or directory
```

It turns out that `qemu` is in `/usr/libexec/` in RHEL, see https://bugzilla.redhat.com/show_bug.cgi?id=1715806#c1

I think the proper fix is to switch to libvirt, the qemu implementation seems to be a bit fragile.

**Comment by larskarlitski** [comment]
> I think the proper fix is to switch to libvirt, the qemu implementation seems to be a bit fragile.

It's not fragile at all: it's the qemu that's used by libvirt, after all. It is not supported to be called directly though. Moving to libvirt might be the right path forward, but could we just use the qemu from `/usr/libexec` for now to get unstuck?

**Comment by ondrejbudai** [comment]
Fixed in #587
